catalogue/webapp/pages/search/images.tsx

```diff
@@ -137,44 +142,46 @@ const ImagesSearchPage: NextPageWithLayout<Props> = ({
         </Space>
       </div>
 
-      {hasNoResults ? (
-        <SearchNoResults
-          query={queryString}
-          hasFilters={hasFilters({
-            filters: filters.map(f => f.id),
-            queryParams: Object.keys(query).map(p => p),
-          })}
-        />
-      ) : (
-        <Wrapper>
-          <Space
-            className="container"
-            v={{ size: 'l', properties: ['padding-bottom'] }}
-          >
-            <PaginationWrapper verticalSpacing="l">
-              <span>{pluralize(images.totalResults, 'result')}</span>
-              <Pagination
-                totalPages={images.totalPages}
-                ariaLabel="Image search pagination"
-                hasDarkBg
-                isHiddenMobile
-              />
-            </PaginationWrapper>
-
-            <main>
-              <ImageEndpointSearchResults images={images.results} />
-            </main>
-
-            <PaginationWrapper verticalSpacing="l" alignRight>
-              <Pagination
-                totalPages={images.totalPages}
-                ariaLabel="Image search pagination"
-                hasDarkBg
-              />
-            </PaginationWrapper>
-          </Space>
-        </Wrapper>
-      )}
+      <Wrapper hasNoResults={hasNoResults}>
+        <Space
+          className="container"
+          v={{ size: 'l', properties: ['padding-bottom'] }}
+        >
+          {hasNoResults ? (
+            <SearchNoResults
+              query={queryString}
+              hasFilters={hasFilters({
+                filters: filters.map(f => f.id),
+                queryParams: Object.keys(query).map(p => p),
```

Comment:
Do we need the `.map(p => p)` here?

Reply:
Nope! and not in works either, I'll remove, thanks!